docs/index.rst

```diff
@@ -343,6 +325,55 @@ function that call a `method` and propagate a `rpc_metadata` dictionary over the
             span.set_meta('my_rpc_method', method)
 
 
+Sampling
+--------
+
+Priority sampling
+~~~~~~~~~~~~~~~~~
+
+Priority sampling consists in deciding if a trace will be kept by using a `priority` attribute that will be propagated
+for distributed traces. Its value gives indication to the Agent and to the backend on how important the trace is.
+
+- 0: Don't keep the trace.
+- 1: The sampler automatically decided to keep the trace.
+- 2: The user asked the keep the trace.
+
+For now, priority sampling is disabled by default. Enabling it ensures that your sampled distributed traces will be complete.
+To enable the priorty sampling::
+
+    tracer.configure(distributed_sampling=True)
+
+Once enabled, the sampler will automatically assign a priority of 0 or 1 to traces, depending on their service and volume.
+
+You can also set this the priority manually to either drop an non-interesting trace or to keep an important one.
+For that, set the `context.sampling_priority` to 0 or 2. It has to be done before any context propagation (fork, RPC calls)
+to be effective::
+
+    context = tracer.get_call_context()
```

Comment:
We have the `tracer.context_provider.active()` API that does the same thing.
 I'd advocate to use that one, since it's the only way to set the current active context. The drawback is that in OpenTracing it will be `scope_manager` instead of `context_provider`. But for the current implementation it doesn't make sense to call the provider "scope manager" because we don't handle scopes.

Reply:
Right I missed this one ; will change!